### DISCUSSION TITLE
How to ScrollTo when pagination changes?

### DISCUSSION THREAD

**Discussion by imnasnainaec** [KevinVandy/material-react-table#975] [Q&A] [answered]
The demo in https://www.material-react-table.com/docs/examples/row-virtualization shows how to scroll to the top of the table whenever sorting changes. I'd like to do the same when pagination changes. However, the `onPaginationChange` property says
> If this function is provided, it will be called when the pagination state changes and you will be expected to manage the state yourself.

Is there any way to execute a function every time the page changes without having to manage the pagination state?

**Comment by KevinVandy** [comment, marked as answer]
no: https://www.material-react-table.com/docs/guides/state-management
